### PR TITLE
regression for vcenter fix

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/9.0/manila.yaml [hunk 1]
```diff
@@ -242,7 +242,7 @@
             cd ${{WORKSPACE}}
             for refspec in ${{MULTI_REFS}}; do
               git fetch ${{GIT_URL}} "$refspec" && git cherry-pick FETCH_HEAD || git status
-              git log --pretty=format:'%h %ad | %s%d [%an]' --graph --date=short
+              git log  --graph --date=short --pretty=format:%h %ad | %s%d [%an]
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml:245]: filter with `head` to avoid mess in the job log

File: servers/partner-ci/9.x/vcenter.yaml [hunk 2]
```diff
@@ -278,7 +278,7 @@
             cd ${{WORKSPACE}}
             for refspec in ${{MULTI_REFS}}; do
               git fetch ${{GIT_URL}} "$refspec" && git cherry-pick FETCH_HEAD || git status
-              git log --pretty=format:'%h %ad | %s%d [%an]' --graph --date=short
+              git log --pretty=format:%h %ad | %s%d [%an]  --graph --date=short
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/vcenter.yaml]: filter with `head` to avoid mess in the job log

File: servers/partner-ci/9.0/manila.yaml (hunk 1 above)
Comment (naumvd95) [reply servers/partner-ci/9.0/manila.yaml:245]: already done in another PR

File: servers/partner-ci/9.x/vcenter.yaml (hunk 2 above)
Comment (naumvd95) [reply servers/partner-ci/9.x/vcenter.yaml]: done

File: servers/partner-ci/9.0/manila.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/manila.yaml:245]: ok

**Review by rkhozinov — APPROVED** [review]
seems good for me.
will be merged

**Pull request merged**